### DISCUSSION TITLE
Conversion of Düren sample set

### DISCUSSION THREAD

**Discussion by rousseldenis** [GrandOrgue/OdfEdit#17] [General]
The conversion was done without errors with 2.2 version.

But no stop can be selected.

**Comment by rousseldenis** [comment]
@eturpault

**Comment by eturpault** [comment]
Hi @rousseldenis, thanks for the report. There is the same issue with the Erfurt Predigerkirche as reported by @oleg68 in the discussion #11. I wrote to Piotr but he didn't answered me. Analysing the Erfurt and Düren sample sets I have understood how the Stop is linked to a Rank in these sample sets, a StopRank object is not mandatory actually. I will implement it for the next OdfEdit v2.3.

**Comment by eturpault** [comment]
In the comment https://github.com/GrandOrgue/ODFEdit/issues/15#issuecomment-1491002509 your can find a v2.3 beta release which fixes this issue and the one of Erfurt are well (I have to fix an issue with the stops noises of these sample sets).

**Reply by rousseldenis** [reply]
Great to hear that! I'll try tonight. Thanks
